lib.php

```diff
@@ -28,6 +28,17 @@
 
 defined('MOODLE_INTERNAL') || die;
 
+/**
+ * Add cron related service status checks
+ *
+ * @return array of check objects
+ */
+function local_envbar_security_checks() : array {
```

Comment:
```suggestion
function local_envbar_security_checks(): array {
```
Should be this (same as previous point)